deepmd/descriptor/se_a.py

```diff
@@ -827,7 +830,12 @@ def _filter(
           # inputs_reshape = tf.reshape(inputs, [-1, shape[1]//4, 4])
           # natom x 4 x outputs_size
           # xyz_scatter_1 = tf.matmul(inputs_reshape, xyz_scatter, transpose_a = True)
-          xyz_scatter_1 = xyz_scatter_1 * (4.0 / shape[1])
+          if self.original_sel is None:
+              # shape[1] = nnei * 4
+              nnei = shape[1] / 4
+          else:
+              nnei = np.sum(self.original_sel)
```

Comment:
I realized that adjusting `sel` again would lose this information.
We need to restore `nnei` forever.